web/implementation-reports/svnmentions.md

```diff
@@ -0,0 +1,109 @@
+# SVNmentions
+
+Implementation Home Page URL: https://github.com/carrvo/SVNmentions
+Version: v1.0
+
+Source Code repo URL(s) (optional): 
+* [ ] 100% open source implementation
+
+Programming Language(s): PHP
+
+Developer(s): carrvo
+
+Implementation Classes (Sender and/or Receiver): Receiver
+
+
+## Receiving
+
+Indicate which type of response the receiver provides:
+
+* [x] HTTP 200 - Receiver synchronously processes the Webmention request (not recommended)
+* [ ] HTTP 201 - Receiver creates a status URL the sender can use to check the status of the Webmention
+* [ ] HTTP 202 - Receiver processes the Webmention asynchronously
+
+Describe the response body (if any) which is returned in the request:
+
+
+### Request Verification (3.2.1)
+
+* [x] Verifies source and target are valid URLs, rejecting with HTTP 400 (MUST)
+* [x] Verifies that target is a valid resource for which the receiver accepts Webmentions, rejecting with HTTP 400 (SHOULD)
```

Comment:
Has some target verification. Has the https://github.com/carrvo/SVNmentions/issues/5 for an additional verification.